.travis.yml

```diff
@@ -1,21 +1,16 @@
-sudo: true
+sudo: required  # create a full VM, not a container
 language: go
 go:
   - 1.9.x
-env:
-  - "PATH=/home/travis/gopath/bin:$PATH"
-before_install:
-  - pwd
-  - ls -l
-install:
-  - ls
-  - curl -L -o /home/travis/gopath/bin/dep https://github.com/golang/dep/releases/download/v0.3.2/dep-linux-amd64
-  - chmod +x /home/travis/gopath/bin/dep
+install: true   # build only with what's in vendor/
```

Comment:
```
before_install
install
before_script
script
```
Separating the commands into these sections is kinda arbitrary and doesn't really make sense for Go projects. I tend to use `before_script` for setting up the test environment, and `script` for actually running the tests, but it could easily all be under `script`. The only important setting is `install: true`, which disables the `install` step, which would've attempted to fetch missed deps that aren't in `vendor/`.